ui/src/components/UI/Tabs.tsx

```diff
@@ -40,7 +39,7 @@ const useTabsStyles = makeStyles((theme) => ({
     },
 
     active: {
-        fontWeight: theme.typography.fontWeightBold,
+        fontWeight: 600,
```

Comment:
isthat intentional checkin?
is it working?

Reply:
its working. theme doesn't seems to work